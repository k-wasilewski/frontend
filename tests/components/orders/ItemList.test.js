import React from 'react';
import renderer from 'react-test-renderer';
import store from "../../../src/redux/store";
import {Provider} from "react-redux";
import {configure, shallow} from 'enzyme';
import Adapter from "enzyme-adapter-react-16";
import ConnectedItemList, { ItemList } from "../../../src/components/orders/ItemList";

describe("ItemList specification", () => {
    it('renders header and ordered list', () => {
        const component = renderer.create(
            <Provider store={store}>
                <ConnectedItemList />
            </Provider>
        );
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();

        expect(tree.children[0].children[0]).toEqual('Lista zamówień')
        expect(tree.children[1].type).toBe('ol')
        expect(tree.children[1].props.start).toBe('1')
    })

    it('fewfewfwef', () => {
        configure({ adapter: new Adapter() });

        const component = shallow(
            <ItemList items = {[]}/>
        )


    })
})