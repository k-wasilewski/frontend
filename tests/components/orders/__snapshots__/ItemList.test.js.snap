// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemList specification renders header and ordered list 1`] = `
<div
  className="main"
>
  <h3>
    Lista zamówień
  </h3>
  <ol
    start="1"
  />
</div>
`;
